src/main/java/com/park/easyrecruit/ejb/UserBean.java

```diff
@@ -22,34 +22,74 @@
  */
 @Stateless
 public class UserBean {
+
     private static final Logger LOG = Logger.getLogger(UserBean.class.getName());
-    
+
     @PersistenceContext
     private EntityManager em;
-    
+
+    public boolean createUser(String username, String email, String hashedPassword, String position) {
+        // Check if username or email is in the User database
+        
+        /*
+        Query query = em.createQuery("SELECT u FROM User u WHERE u.username = uname" OR u.email = mail)
```

Comment:
try this first and add the or condition for email later if this works.
```suggestion
        Query query = em.createQuery("SELECT u FROM User u WHERE u.username = :uname").setParameter("uname", username);
```

Reply:
I've changed those lines and tested it